### PR TITLE
Remove errors from phone numbers guidance

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes [#2022](https://github.com/alphagov/govuk-design-system/issues/2022).

Updates the ['links' section of our telephone numbers guidance](https://design-system.service.gov.uk/patterns/telephone-numbers/#do-not-display-telephone-numbers-as-links-on-devices-that-cannot-make-calls).

### Changes
Current guidance says most:

- desktop machines do not support calls - this is untrue so we're changing it
- modern mobile browsers detect phone numbers and display them as links - as above

Also, when users click on linked numbers, it can trigger a Skype setup process or the like. So we're also adding a sentence to tell users to bear this in mind.

### CODE REVIEW

**Review by owenatgov — APPROVED** [review]
I'm personally happy with this. Probably worth getting a second approval from the other, more content averse reviewers.

**Comment by EoinShaughnessy** [comment]
@owenatgov Thanks Owen, appreciate the speedy review!

**Comment by davidc-gds** [comment]
Hey @EoinShaughnessy! I'm not sure what's needed for a review, but I might hop in to second-guess the removal of that second line.

For this line:

> most modern mobile browsers detect phone numbers and display them as links

Should we just keep it in and update it to be more accurate?

> some modern mobile browsers detect phone numbers and display them as links

Not sure if that'll just add more ambiguity about what to do - but hey, this is a pattern not a component.

**Comment by EoinShaughnessy** [comment]
@davidc-gds Great suggestion, thanks! Will add in. 👍🏻

**Review by davidc-gds — APPROVED** [review]
Looks good now, thanks Eoin!

**Comment by EoinShaughnessy** [comment]
@calvin-lau-sig7 Does the update look ok to you? :)

File: src/patterns/telephone-numbers/index.md.njk
Comment (calvin-lau-sig7) [inline src/patterns/telephone-numbers/index.md.njk:85]: On my first read, this para looked a tad patronising. It took me a while to get the context for this in the issue, but how about front-loading the 'user journey' message?

I'm also at a bit of a loss on how to be clear about the Skype situation without calling them out specifically. I think we try to avoid naming names in this 'layer' of guidance. 😄 

```suggestion
When looking at the user journey for your service, remember that telephone numbers as links might behave in unexpected ways for the user. For example, some browsers and operating systems will automatically start a setup process to install bundled software if the user has not set a default app to handle `tel:` links.
```

Comment (EoinShaughnessy) [reply src/patterns/telephone-numbers/index.md.njk:85]: @calvin-lau-sig7 That looks good, thanks! Might just add the _lightest_ of tweaks...

```suggestion
When you look at the user journey for your service, remember that telephone numbers as links might behave in unexpected ways for the user. For example, unless the user sets a default app to handle `tel:` links, some browsers and operating systems will automatically start a setup process.
```

Comment (calvin-lau-sig7) [reply src/patterns/telephone-numbers/index.md.njk]: Looks good! Reading it, it feels more neutral and balance than what I'd wrote.

Approving now — feel free to commit and merge.

**Review by calvin-lau-sig7 — APPROVED** [review]

**Pull request merged**
